source/_includes/integrations/config_flow.md

```diff
@@ -3,7 +3,7 @@
 
 ## Configuration
 
-To add the **{{ name }}** integration to your Home Assistant instance, use this My button:
+To add the **{{ name }}** {{ page.ha_integration_type | default: "integration" }} to your Home Assistant instance, use this My button:
```

Comment:
Wouldn't this pull also other types, not just helpers? 
<https://developers.home-assistant.io/docs/creating_integration_manifest/#integration-type>

Reply:
You're right, but base it on existing behavior: https://github.com/home-assistant/home-assistant.io/blob/651c0ae4a2c0b7afbda278d7d15a889aadb3f49a/source/_includes/asides/component_navigation.html#L33

But it doesn't seem ideal for some types, in both places... do you think it's better to use an IF in this PR just for helpers or leave it as is and close this PR?

Reply:
Right. Forgot we had it there, too. On the other hand, this filter is only applied in the config flow. For example, I assume the config flow wouldn't be used for entities?

Reply:
For entities and system I don't think there will be a use case. For device, hub and service, the code as is is acceptable. However, for virtual and hardware I think it can get a little strange (especially for virtual).